test/library/idpdeployment.go

```diff
@@ -136,7 +136,7 @@ func deployPod(
 	_, err = clients.CoreV1().Services(namespace).Create(testContext, svcTemplate(httpPort, httpsPort), metav1.CreateOptions{})
 	require.NoError(t, err)
 
-	timeLimitedCtx, cancel := context.WithTimeout(testContext, 5*time.Minute)
+	timeLimitedCtx, cancel := context.WithTimeout(testContext, 10*time.Minute)
```

Comment:
@benluddy FYI since your last review, I decided to double this because I was noticing several flakes timing out on this.

Reply:
Is there some misconfiguration in the keycloak deployment that is causing it to take so long? I don't know if it needs to be addressed immediately, but it would be a shame to needlessly pay an extra ~five minutes of test runtime (and test infrastructure costs) forever.